routes/routes.js

```diff
@@ -38,5 +38,31 @@ router.post('/user/answers/question/:questionNumber', (req, res) => {
     })
 })
 
+router.get('/user/answers/question/:questionNumber', (req, res) => {  
+    let pageSize = 3; // TODO: define it in more stadarized way 
+    
+    User.findOne({ 
+        where: {id: req.query.userId}
+    }).then(user => {
+        
+        UserAnswers.findAll({
+            where: {question_number: req.params.questionNumber, user_id: user.id},
+            order: [
+                ['created_at', 'DESC']
+            ],
+            limit: pageSize
+        }).then(userAnswers => {
+            return res.status(200).json({
+                previousTimingMs: userAnswers.map(userAnswer => userAnswer.elapsed_time_ms)
+            })
+        }).catch(error => {
+            console.log(error);
+            return res.status(400).json(error.errors) // TODO: handle better error messages
```

Comment:
Since we don't know what the cause of the error was, we should probably use code 500 as it could be a bug on the server.